plotting.py

```diff
@@ -455,7 +459,7 @@ def plot_profile(datasets,
     # convert to a list
     if isinstance(fields,str):
         fields = [fields,]
-    if isinstance(times,str):
+    if isinstance(times,(str,int,float,np.number)):
```

Comment:
I think the documentation needs to be updated to indicate that times can be a list or a str/int/float